### ISSUE TITLE
Encoding in scaffolded Items

### ISSUE THREAD

**Issue by BFISoft** [aspnet/Scaffolding#144]
When creating a scaffolded item in MVC6 project there seems to be an inconsistency with file encodings.

Normal files (without scaffolding) are created as UTF-8
Scaffolded controllers and views are created as ANSI

Which means that text containing äüöü in _Layout is displayed correctly but in the scaffolded view the same text shows some weird symbols.

Microsoft Visual Studio Community 2015
Version 14.0.24627.00 Update 1 RC
Microsoft .NET Framework
Version 4.6.01040

ASP.NET and Web Tools 2015 (RC) 14.1.11116.0
ASP.NET and Web Tools 2015 (RC)

ASP.NET Web Frameworks and Tools 2012.2 4.1.40928.0
For additional information, visit http://go.microsoft.com/fwlink/?LinkID=309563

ASP.NET Web Frameworks and Tools 2013 5.2.30624.0
For additional information, visit http://www.asp.net/

**Comment by blanceok** [comment]
I've found the cause, and the solution.
Now I have attached the documents I have changed.
Path: C:\Users\"UserName"\.dnx\packages\Microsoft.Extensions.CodeGenerators.Mvc\1.0.0-rc1-final\Templates

Unable to upload the.Zip file, the file can not be changed.

Change method:
Microsoft.Framework.CodeGenerators.Mvc templates have problems:
1、*.cshtml's archive encoding is not UTF8,Cause some language can not be normal, such as Chinese.
2、Server Model data validation, error message can not be normal display.The reason shows that the `<span/>` has a problem with the error message, and the `<span></span>` can be displayed as normal.
